### ISSUE TITLE
Bug when placing moving landmark in target viewer

### ISSUE THREAD

**Issue by aschampion** [saalfeldlab/bigwarp#53]
When placing a target *and* a moving landmark in the *target* viewer with <kbd>SPACE</kbd> + click and then <kbd>SHIFT</kbd> + click, sometimes instead one ends up with two landmarks, named the same, one with the target location set (and the moving infinity) and one with the moving location set (and the target infinity). The expected behavior is that the <kbd>SHIFT</kbd> + click would fill in the moving location for the first row, just as if one had clicked in the moving viewer.

**Comment by bogovicj** [comment]
Agreed.

**Comment by bogovicj** [comment]
98417ad  should address this issue.

Putting my thoughts here as I play around with the consequences of this more.

Options I considered to enable adding moving landmarks in the fixed viewer (and vice versa):
1) <kbd>SHIFT</kbd>+ click; when there is an "incomplete" row in the landmark table (what we talked about in person today, @aschampion )
2) <kbd>SHIFT</kbd>+ <kbd>CTRL </kbd> + click; at any time

Decided to go with the latter because the former makes an action depend on context, and it confused me in my initial usability tests.  The latter also allows one to put points in the "other" space before you start a landmark pair with a "normal" <kbd>SPACE </kbd> + click.  

Downside of clicking in the other space first is that a landmark marker doesn't appear in the viewer window, which is unnerving ( *"I just clicked there...why didn't anything happen!?"* ), but arguably the correct behavior.  Still think it's the way to go, but I intend to do a little more usability testing before merging this into master.
